contexts/security-v3-unstable.jsonld

```diff
@@ -88,85 +103,113 @@
-              "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
-            },
-            "domain": "https://w3id.org/security#domain",
-            "nonce": "https://w3id.org/security#nonce",
-            "proofPurpose": {
-              "@id": "https://w3id.org/security#proofPurpose",
-              "@type": "@vocab",
-              "@context": {
-                "@version": 1.1,
-                "@protected": true,
-                "id": "@id",
-                "type": "@type",
-                "sec": "https://w3id.org/security#",
-                "assertionMethod": {
-                  "@id": "https://w3id.org/security#assertionMethod",
-                  "@type": "@id",
-                  "@container": "@set"
-                },
-                "authentication": {
-                  "@id": "https://w3id.org/security#authenticationMethod",
-                  "@type": "@id",
-                  "@container": "@set"
-                }
+          "@version": 1.1,
+          "@protected": true,
+          "id": "@id",
+          "type": "@type",
+          "challenge": "https://w3id.org/security#challenge",
+          "created": {
+            "@id": "http://purl.org/dc/terms/created",
+            "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+          },
+          "domain": "https://w3id.org/security#domain",
+          "nonce": "https://w3id.org/security#nonce",
+          "proofPurpose": {
+            "@id": "https://w3id.org/security#proofPurpose",
+            "@type": "@vocab",
+            "@context": {
+              "@version": 1.1,
+              "@protected": true,
+              "id": "@id",
+              "type": "@type",
+              "assertionMethod": {
+                "@id": "https://w3id.org/security#assertionMethod",
+                "@type": "@id",
+                "@container": "@set"
+              },
+              "authentication": {
+                "@id": "https://w3id.org/security#authenticationMethod",
+                "@type": "@id",
+                "@container": "@set"
+              },
+              "capabilityInvocation": {
+                "@id": "https://w3id.org/security#capabilityInvocationMethod",
+                "@type": "@id",
+                "@container": "@set"
+              },
+              "capabilityDelegation": {
+                "@id": "https://w3id.org/security#capabilityDelegationMethod",
+                "@type": "@id",
+                "@container": "@set"
+              },
+              "keyAgreement": {
+                "@id": "https://w3id.org/security#keyAgreementMethod",
+                "@type": "@id",
+                "@container": "@set"
               }
-            },
-            "proofValue": "https://w3id.org/security#proofValue",
-            "verificationMethod": {
-              "@id": "https://w3id.org/security#verificationMethod",
-              "@type": "@id"
             }
+          },
+          "proofValue": "https://w3id.org/security#proofValue",
```

Comment:
Should this have ` "@type": "https://w3id.org/security#multibase"` or not? Not sure what `BbsBlsSignature2020` uses for `proofValue`. But `Ed25519Signature2020` should have `proofValue` defined that way. This implies not defining it at the top level so that each suite can define it separately.

Reply:
@dlongley interesting will review but on first though multi-base sounds like a good idea.

Reply:
Currently it's not a multibase formatted value, but I'm in favor of switching to that. Right now we just use a base64 (not base64url) encoded value.

For now I've removed the top level definition so that we can define these differently which should fix the problem @peacekeeper discovered. I think this is the appropriate way to handle this because the older suites do not use multibase either which would create conflict between the older definitions and the newer ones unless we property scope them.

If anyone would prefer to see us do it differently or has some other considerations that should be addressed please open an issue and we can decide there.